docs/en/enterprise-edition/rn/look-ahead-planned-updates-prisma-cloud/look-ahead-secure-the-infrastructure.adoc

```diff
@@ -313,59 +198,23 @@ config from iam where dest.cloud.type = 'GCP' AND action.name contains all('run.
 |*Service*
 |*API Details*
 
-|*Amazon S3*
+|*AWS WAF*
 
-tt:[24.7.3]
-//RLP-142168
+tt:[24.8.1]
+//RLP-134184
 
-|*aws-s3api-get-bucket-acl*
+|*aws-waf-v2-global-rule-group*
 
 Additional permission required:
 
-* `s3:GetAccelerateConfiguration`
-
-The Security Audit role includes the permission.
-
-Also, this API has been updated to include the following new field in the resource JSON:
-
-* `bucketAccelerateConfiguration`
-
-|*AWS Application Migration Service*
-//RLP-145208
-
-|*aws-mgn-replication-configuration-template*
-
-Additional permission required:
+* `wafv2:ListRuleGroups`
+* `wafv2:GetRuleGroup`
 
-* `mgn:DescribeReplicationConfigurationTemplates`
-
-The Security Audit role does not include the permission. You must manually add the above permission to the CFT template to enable it.
-
-
-|*Amazon EMR*
-//RLP-145210 
-
-|*aws-emr-serverless-application*
-
-Additional permissions required:
-
-* `emr-serverless:ListApplications`
-* `emr-serverless:GetApplication`
-
-The Security Audit role includes the permissions. 
+The Security Audit role includes the `wafv2:ListRuleGroups` permission. wafv2:GetRuleGroup is NOT present in Security Audit Policy
```

Comment:
Rewrite as follows:
The Security Audit role only includes the `wafv2:ListRuleGroups` permission. You must manually add the `wafv2:GetRuleGroup` permission to the CFT template to enable it.